extensions/amp-autocomplete/0.1/test/test-amp-autocomplete.js

```diff
@@ -661,22 +664,23 @@ describes.realWin(
           expect(impl.container_.children.length).to.equal(3);
           expect(resetSpy).to.have.been.calledOnce;
           impl.toggleResults_(true);
-          expect(impl.resultsShowing_()).to.be.true;
+          expect(impl.areResultsDisplayed_()).to.be.true;
           return impl.keyDownHandler_(event);
         })
         .then(() => {
           expect(displayInputSpy).to.have.been.calledOnce;
           expect(resetSpy).to.have.been.calledTwice;
           expect(toggleResultsSpy).to.have.been.calledWith(false);
-          expect(impl.resultsShowing_()).to.be.false;
+          expect(impl.areResultsDisplayed_()).to.be.false;
         });
     });
 
     it('should call keyDownHandler_() on Tab', () => {
-      const event = {key: Keys.TAB};
+      const event = {key: Keys.TAB, preventDefault: () => {}};
```

Comment:
Does it make sense to spy `preventDefault`?

Reply:
Yes, we should know if the `Tab` is fully consumed by the listener.